apis/extension/slo_controller_config.go

```diff
@@ -35,15 +35,15 @@ const (
 // +k8s:deepcopy-gen=true
 type NodeCfgProfile struct {
 	// like ID for different nodeSelector; it's useful for console so that we can modify nodeCfg or nodeSelector by name
-	Name string `json:"name,omitempty"`
+	Name string `json:"name,omitempty" validate:"required"`
```

Comment:
this field is an internal field managed by console, maybe we don't need to check it.

Reply:
enabled via a command line argument